docs/sources/datasources/azure-monitor/query-editor/index.md

```diff
@@ -109,12 +112,13 @@ You can also perform complex analysis of Logs data by using KQL.
 **To create a Logs query:**
 
 1. In a Grafana panel, select the **Azure Monitor** data source.
-1. Select the **Logs** service.
-1. Select a resource to query. Multiple resources can be selected as long as they are of the same type.
+2. Select the **Logs** service.
+3. Select a resource to query. Multiple resources can be selected as long as they are of the same type.
 
    Alternatively, you can dynamically query all resources under a single resource group or subscription.
+   **Note:** If a timespan is specified in the query the overlap of the timespan between the query and the dashboard will be used as the query timespan. See the [API documentation for details.](https://learn.microsoft.com/en-us/rest/api/loganalytics/dataaccess/query/get?tabs=HTTP#uri-parameters)
```

Comment:
Nit: add a comma after query so it reads `If a timespan is specified in the query, the overlap...`